### ISSUE TITLE
Proton doesn't work at all...

### ISSUE THREAD

**Issue by daxime** [ValveSoftware/Proton#3295]
## Symptoms
I've been using Proton for a year. At first I was on Ubuntu 18.04.1, then I updated to 18.10 and 19.04. Proton worked very well.

I wanted to test Pop_OS! 19.04 but Vulkan never worked on it and several games didn't work (sometimes the luncher of the game starts, but not the game, or with very big slowdown).

2 days ago I reinstalled Ubuntu 18.04.3, Vulkan works very well but no Windows games work (Proton 4.11-9). Even games that worked without any problem with Proton 4.11-X, no longer work...

I started Steam via the terminal to get the error messages, there are none even when I start a game... I have no idea what the source of the problem is... When I start a game, my nickname turns green and 2-3 seconds later it turns blue again. Nothing else is going on. Native Linux games work very well.

I tried to add `PROTON_LOG=1 %command%`, but it don't generate the log file ...

I don't know if it matters, but my game library is on a secondary hard drive (for not having to re-download the games when i change my distro).

## System Information
- CPU: Ryzen 5 2600
- GPU: AMD HD 7850
- Driver/LLVM version: 4.5 Mesa 19.1.2 - padoka PPA / LLVM-8
- Kernel version: 5.0.0-37-generic
- Proton version: 4.11-9

## Reproduction
Start any Windows game.

**Comment by kisak-valve** [comment]
Hello @daxime, what filesystem are you using with the second hard drive?

Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. What I'm interested in is if you have a working vulkan driver stack.

You have a Southern Island (SI) generation video card, and Linux uses the `radeon` kernel by default. The `radeon` kernel module is not compatible with Vulkan. Please give https://github.com/ValveSoftware/Proton/wiki/For-AMD-users-having-issues-with-non-OpenGL-games a read if you're not familiar with switching over to the `amdgpu` kernel module.

**Comment by daxime** [comment]
Hello :)
The second HDD use Ext4. I have modified my GRUB to use AMDGPU

```
08:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Pitcairn PRO [Radeon HD 7850 / R7 265 / R9 270 1024SP]
	Subsystem: PC Partner Limited / Sapphire Technology Radeon HD 7850 2GB GDDR5 DVI-I/DVI-D/HDMI/DP
	Kernel driver in use: amdgpu
	Kernel modules: radeon, amdgpu
```
Sys. info : https://gist.github.com/daxime/08e564c5affada898314203a6f390113

**Comment by Simon-Swenson-8351** [comment]
Hello. I was also having a similar problem. My PC is also a Ryzen PC (Ryzen 3600x, RTX 5700 XT). All of the other related issues I have found thus far have been related to the filesystem type. People were trying to launch games from an NTFS partition. However, my app folder is on my boot ext4 drive. Depending on the game, I get an error: "wine: cannot find <EXE FILE>" or "pressure-vessel-with-lock: Failed to execute child process "/home/simon/.local/share/Steam/steamapps/common/DARK SOULS REMASTERED/DarkSoulsRemastered.exe" (No such file or directory)".

These error messages went away if I changed the proton version used: right-click the game -> Properties -> Force the use of a specific Steam Play compatibility tool -> Proton 4.11-9.

The root of the problem seemed to be that, when I let steam decide which proton version it tried to use, it was defaulting to Steam Linux Runtime, when the game is not a native Linux app. This was a user error, as when I enabled Steam Play for other (non-supported titles) the default option was "Steam Linux Runtime" and not some Proton version.

**Comment by daxime** [comment]
This was my previous problem. When I started a game, it never found the.exe file. When I opened the game folder, I was surprised to see that it was almost empty... When checking the proton settings I saw "Steam Linux Runtime", I selected 4.11-9, and I had to re-download about 150GB of games...

But from there, no games start and i have no error... I'll try to select another version of proton.

**Edit:** It was de appmanifest. I had deleted the proton folder, so Steam thought there was proton installed when there wasn't... I have removed the corresponding manifest file, so problem solved.
